packages/client/src/containers/CollectionsPage/CollectionsPage.Container.js

```diff
@@ -1,12 +1,91 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import ProductList from '../../components/ProductList/ProductList.component';
-import DropDownViewList from '../../components/ProductListDropDown/DropDownViewList/DropDownViewList.component';
+import './CollectionsPage.style.css';
+import DropDownView from '../../components/CategoriesListDropDown/CategoriesListDropDown.component';
+import { apiURL } from '../../apiURL';
 
 export const CollectionsPage = () => {
+  const [products, setProducts] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [onSelectedCategory, setOnSelectedCategory] = useState('');
+  const [onSelectedSortBy, setOnSelectedSortBy] = useState('');
+  // const [onSelectedAllFilter, setOnSelectedAllFilter] = useState('');
+
+  const fetchProducts = () => {
+    setIsLoading(true);
+    fetch(`${apiURL()}/products/`)
+      .then((res) => res.json())
+      .then((item) => {
+        setProducts(item);
+      })
+      .then(() => {
+        setIsLoading(false);
+      });
+  };
+  useEffect(() => {
+    fetchProducts();
+  }, []);
+
+  const [category, setCategoy] = useState([]);
+  const fetchCategories = () => {
+    fetch(`${apiURL()}/categories/`)
+      .then((res) => res.json())
+      .then((item) => {
+        setCategoy(item);
+      });
+  };
+  useEffect(() => {
+    fetchCategories();
+  }, []);
+
+  const categories = category.map(
+    (element) =>
+      element.name.charAt(0).toUpperCase() +
+      element.name.slice(1).toLowerCase(),
+  );
```

Comment:
```suggestion
const [categories, setCategories] = useState([]);

useEffect(() => {
  const fetchCategories = () => {
    fetch(`${apiURL()}/categories`)
      .then((res) => res.json())
      .then((items) => {
        setCategories(items.map(
          (item) =>
            item.name.charAt(0).toUpperCase() +
            item.name.slice(1).toLowerCase(),
        ));
      });
  };

  fetchCategories();
}, []);
```

You'll have a problem here though that `categories` will be a list of strings, how will you get access to the ID?